docs/source/reference/config-sudo.md

```diff
@@ -120,6 +120,11 @@ the shadow password database.
 
 ### Shadow group (Linux)
 
+**Note:** On Fedora based distributions there is no clear way to configure
+the PAM database to allow sufficient access for autenticating with the target user's password
+from jupyterhub.As a workaround we recommend use an alternative authentication method, e.g.
+[sshauthenticator](https://github.com/andreas-h/sshauthenticator).
+
```

Comment:
```suggestion
**Note:** On Fedora based distributions there is no clear way to configure
the PAM database to allow sufficient access for autenticating with the target user's password
from JupyterHub. As a workaround we recommend use an alternative authentication method, e.g.
[sshauthenticator](https://github.com/andreas-h/sshauthenticator).

```

Formatting detail. I'm don't understand the details about this enough to review the actual content though =/

Reply:
Also a typo in `autenticating`.

I'm not sure about recommending ssh as an alternative though, maybe just link to https://github.com/jupyterhub/jupyterhub/wiki/Authenticators instead?